clang/test/Analysis/out-of-bounds-notes.c

```diff
@@ -0,0 +1,128 @@
+// RUN: %clang_analyze_cc1 -Wno-array-bounds -analyzer-output=text        \
+// RUN:     -analyzer-checker=core,alpha.security.ArrayBoundV2,unix.Malloc,alpha.security.taint,debug.ExprInspection -verify %s
+
+int array[10];
+
+void clang_analyzer_value(unsigned arg);
```

Comment:
I'm worried about implicit conversions if at the call site we have a signed int, that could affect what range we display.
I'm not sure if this is true or not.

Reply:
I'm not actually using `clang_analyzer_value()`, I just forgot to remove it after some temporary changes I did for debugging something.